index.d.ts

```diff
@@ -0,0 +1,80 @@
+import { EventEmitter } from "events";
+import { User, Guild, Channel, Client, ClientOptions } from "eris";
+
+declare namespace ErisSharder {
+  interface Sharder extends EventEmitter {
+    shardCount: number;
+    firstShardID: number;
+    lastShardID: number;
+    clusterCount: number;
+    clusterTimeout: number;
+    token: string | false;
+    clusters: Map<>;
+    workers: Map<>;
+    queue: ErisSharderQueue;
+    eris: Client;
+    options: {
+      stats: ErisSharderStats;
+    };
+    statsInterval: number;
+    mainFile: unknown;
+    name: string;
+    guildsPerShard: number;
+    webhooks: {
+      cluster?: unknown;
+      shard?: unknown;
+    };
+    debug: boolean;
+    clientOptions: ClientOptions;
+		callbacks: Map<>;
+		isMaster(): boolean;
+		startStats(): void;
+		executeStats(clusters: unknown[], start: number): void;
+		start(clusterID: number): void;
+		launch(): void;
+		chunk(shards: unknown, clusterCount: number): unknown[]
+		connectShards(): void;
+		sendWebhook(type: string, embed): void;
+		printLogo(): void;
+		restartCluster(worker: unknown, code: unknown): void;
```

Comment:
nodejs worker (should be a type for this somewhere), number

Reply:
Couldn't find the nodejs worker type so i kept it as unknown for now.